server/src/utils/vm/wollok.ts

```diff
@@ -73,4 +73,8 @@ export const projectFQN = (node: Entity): string => {
   if (!rootPath) return node.fullyQualifiedName
   const rootFQN = rootPath.replaceAll(path.sep, '.')
   return node.fullyQualifiedName?.replaceAll(rootFQN + '.', '') ?? ''
-}
\ No newline at end of file
+}
+
+export const referenceOf = <T extends Node>(aNode: T) => (anotherNode: Node): anotherNode is Reference<T>  => {
+  return anotherNode.is(Reference) && anotherNode.target === aNode
```

Comment:
```suggestion
export const referenceOf = <T extends Node>(targetNode: T) => (referenceNode: Node): referenceNode is Reference<T>  => {
  return referenceNode.is(Reference) && referenceNode.target === targetNode
```

- Yo la renombraría a `isTargetOf`
- Y supongo que esto es algo que queremos en https://github.com/uqbar-project/wollok-ts/issues/171

Reply:
Pero `isTargetOf` no se referiria al nodo que esta siendo targetteado?

Que te parece `isTargetting`?

Reply:
Ahh SII ! O `targetTo` ?

Reply:
Me gusta, lo voy a dejar como `targettingAt`